dist/stat_data.py

```diff
@@ -776,3 +780,18 @@ def __init__(self, name, desc, flags=''):
 ]
 
 join_stats = sorted(join_stats, key=attrgetter('desc'))
+
+##########################################
+# Session operation statistics
+##########################################
+session_op_stats = [
```

Comment:
I think that the stats are specifically for the r/w ops done in a session, it would be good to distinguish those with this name than keeping a generic name. Do you prefer `session_stats`?